### PR TITLE
PLAT-1124 Fix style of EmfFormFrame

### PR DESCRIPTION
**Pull request by daniel-klose**
Left: EmfFormFrame Right: EmfFormPopup
image

### CODE REVIEW

File: platform-emf/src/main/java/com/softicar/platform/emf/form/frame/EmfFormFrameHeader.java [hunk 1]
```diff
@@ -14,7 +14,15 @@ public EmfFormFrameHeader() {
 	public EmfFormFrameHeader setCaption(IDisplayString entityTitle, IDisplayString operationTitle) {
 
 		removeChildren();
-		appendText(entityTitle.concat(" - ").concat(operationTitle).toString());
+
+		var captionDiv = appendChild(new DomDiv());
```
Comment (alexander-schmidt) [inline platform-emf/src/main/java/com/softicar/platform/emf/form/frame/EmfFormFrameHeader.java:18]: I think we should have a `private class EmfFormFrameCaption` here - at least that's what we usually do when we have a special CSS class.

File: platform-emf/src/main/java/com/softicar/platform/emf/form/frame/EmfFormFrameHeader.java [hunk 2]
```diff
@@ -14,7 +14,15 @@ public EmfFormFrameHeader() {
...
+		captionDiv.setCssClass(EmfCssClasses.EMF_FORM_FRAME_CAPTION);
+		captionDiv.appendText(entityTitle);
+
+		var subCaptionDiv = appendChild(new DomDiv());
```
Comment (alexander-schmidt) [inline platform-emf/src/main/java/com/softicar/platform/emf/form/frame/EmfFormFrameHeader.java:22]: same as above

File: platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css [hunk 3]
```diff
@@ -14,16 +14,33 @@
 	border-radius: var(--BORDER_RADIUS);
 	box-shadow: var(--BOX_SHADOW_OVERLAY);
 	display: inline-grid;
+	grid-auto-columns: max-content;
+	align-self: start;
 	outline-style: none;
+	gap: 0px;
```
Comment (alexander-schmidt) [inline platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css:20]: Why is this necessary, all of a sudden, and why won't things look gapless now?

File: platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css [hunk 4]
```diff
@@ -78,9 +95,7 @@
 	grid-column: 1/3;
 }
 
-.EmfFormCommonActionsDiv {
-	gap: 4px;
-	display: flex;
+.DomActionBar.EmfFormCommonActionsDiv {
```
Comment (alexander-schmidt) [inline platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css:98]: Do handcrafted context menus still look like before now?

**Review by alexander-schmidt — CHANGES_REQUESTED** [review]

File: platform-emf/src/main/java/com/softicar/platform/emf/form/frame/EmfFormFrameHeader.java (hunk 1 above)
Comment (daniel-klose) [reply platform-emf/src/main/java/com/softicar/platform/emf/form/frame/EmfFormFrameHeader.java:18]: hm, i looked in `DomPopup` and there is essentially the same thing as this, just a little bit different
image

File: platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css (hunk 4 above)
Comment (daniel-klose) [reply platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css:98]: Well, EmfFormCommonActionsDiv extends DomActionBar, so nothing should have changed
If you mean handcrafted context menus like in the module classes page, then yes, everything is the same

File: platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css (hunk 3 above)
Comment (daniel-klose) [reply platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css:20]: This is necessary to make the EmfFormFrame have the same dimensions and gaps as an EmfFormPopup.
EmfFormFrame has 2 Components, to the 5px gap would make a frame seem bigger. See screenshot
image

File: platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css (hunk 4 above)
Comment (alexander-schmidt) [reply platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css:98]: Ooops, misread the class name. Never mind.

File: platform-emf/src/main/java/com/softicar/platform/emf/form/frame/EmfFormFrameHeader.java (hunk 1 above)
Comment (alexander-schmidt) [reply platform-emf/src/main/java/com/softicar/platform/emf/form/frame/EmfFormFrameHeader.java:18]: well, whatever

**Review by alexander-schmidt — APPROVED** [review]
